DECISION-MAKING.md

```diff
@@ -0,0 +1,79 @@
+      evaluate it.
+   1. The PR features at least one [Review that Approves](https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews) 
+      the PR from a Contributor that is not an author of the PR. The review 
+      needs to be made after the last commit in the PR (equivalent to 
+      [Stale review dismissal](https://help.github.com/articles/enabling-required-reviews-for-pull-requests/)
+      option on GitHub).
+   1. Does not feature any [Reviews that Request changes](https://help.github.com/articles/about-required-reviews-for-pull-requests/).
+   1. Does not feature "WIP" in the title (Work in Progress).
+   1. Passes all automated tests.
+1. Any Contributor can Review a PR and Request changes. If a Contributor 
+   Request changes they need to provide an explanation what changes 
+   should be added and justification of their importance. Reviews requesting 
+   changes can also be used to request more time to review a PR.
+1. A Contributor that Requested changes can Dismiss their own review or Approve 
+   changes added by the Contributor who opened the PR.
+1. If the author of a PR and Contributor who provided Review that Requests 
+   changes cannot find a solution that would lead to the Contributor dismissing 
+   their review or accepting the changes the Review can be Dismissed with a 
+   vote. Rules governing voting:
+   1. A Vote can be triggered by any Contributor, but only after 5 working days 
+      from the time a Review Requesting Changes has been raised and in case a 
+      Vote has been triggered previously no sooner than 15 working days since 
+      its conclusion.
+   1. Only Contributors can vote, each contributor gets one vote.
+   1. A Vote ends after 5 working days or when all Contributors have voted 
+      (whichever comes first).
+   1. A Vote freezes the PR - no new commits or Reviews Requesting changes can 
+      be added to it while a vote is ongoing. If a commit is accidentally made 
+      during that period it should be reverted.
+   1. The quorum for a Vote is 30% of all Contributors.
+   1. The outcome of the vote is decided based on a simple majority.
+
+## Comments
+
+1. There are no restrictions on how the content of the PR is prepared. For 
+   example it is perfectly fine for a PR to consist of content developed by a 
+   group of experts over an extended period of time via in person meetings and 
+   online collaborations using a Google Document.
+1. To facilitate triaging of incoming PR you can subscribe to 
+   notifications for new PRs proposing changes to specific files. To do this
+   add your Github name next to the file you want to subscribe to in the 
+   [CODEOWNERS](CODEOWNERS). This way you will be ask to review each relevant
+   PR. Please mind that lack of your review will not prevent the PR from being
+   merged so if you think the PR needs you attention review it promptly or
```

Comment:
Just a syntax+politeness comment here: "...you attention review..." => "...your attention, please review..."

Reply:
Thanks!